model-optimizer/mo/utils/summarize_graph.py

```diff
@@ -70,9 +70,10 @@ def summarize_graph(graph_def):
     if argv.input_model and argv.saved_model_dir:
         print("[ ERROR ] Both keys were provided --input_model and --input_dir. Please, provide only one of them")
         sys.exit(1)
-    graph_def, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
-                                     checkpoint=argv.input_checkpoint,
-                                     model_dir=argv.saved_model_dir, saved_model_tags=argv.saved_model_tags)
+    tags = argv.saved_model_tags.split(",")
+    graph_def, _, _ = load_tf_graph_def(graph_file_name=argv.input_model, is_binary=not argv.text,
```

Comment:
Can we add some unit test for this case? May be extend TestingSummarizeGraph?

Reply:
It is hard to write: tags are used only for saved_model format, but this format requires a special structure directory for model, not a pb-file. And more, this fix does processing of the command line parameter `--saved_model_tags` according to the help for this parameter: the value of this parameter is split into a list of strings by commas.